src/Mount.hpp

```diff
@@ -343,6 +336,11 @@ class Mount
     // Set the speed of the given motor
     void setSpeed(StepperAxis which, float speedDegsPerSec);
 
+#if (USE_RA_END_SWITCH == 1) || (USE_DEC_END_SWITCH == 1)
+    void setupEndSwitches();
+    void processEndSwitchState();
```

Comment:
`processEndSwitchState` isn't used (and not implemented)

Reply:
Removed.